bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -46,6 +47,7 @@
 
 		<label>Zone</label>
 		<description>A zone of a home</description>
+		<semantic-equipment-tag>Thermostat</semantic-equipment-tag>
```

Comment:
Just question, should it be `Thermostat` or `Zone`?

Reply:
Good observation. In tado there is one Thermostat per Zone so these are synonymous. But Thermostat gives a better overview of the actual functionality.